Gemfile

```diff
@@ -69,6 +69,7 @@ source 'https://rubygems.org' do
   end
 
   group :development do
+    gem 'letter_opener'
```

Comment:
Can we remove this in favour of a mailer preview instead? This follows with the rest of our emails.